CMakeLists.txt

```diff
@@ -125,14 +125,14 @@ elseif(NOT CMAKE_CROSSCOMPILING)
 endif()
 
 #-----------------------------------------------------------------------
-# GetDownloadSpec
+# GetDownloadClause
 #
 # Generates the ExternalProject_Add() parameters needed to download
 # a git repository and returns them in the VAR parameter. Returns an
 # empty string if the DOWNLOAD option is false.
 #-----------------------------------------------------------------------
 
-function(GetDownloadSpec VAR _url _tag)
+function(GetDownloadClause VAR _url _tag)
   if(DOWNLOAD)
     set(_spec
```

Comment:
Should the variable name also be changed to _clause instead of _spec to improve readability?

Reply:
Yes, it should. Good catch.